app-catalog/src/components/charts/List.tsx

```diff
@@ -318,7 +383,7 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
             size="large"
             shape="rounded"
             page={page}
-            count={Math.floor(chartsTotalCount / PAGE_OFFSET_COUNT_FOR_CHARTS)}
+            count={Math.ceil(chartsTotalCount / PAGE_OFFSET_COUNT_FOR_CHARTS)}
```

Comment:
This bug fix could be in a separate commit.

Reply:
new pr needed

Reply:
Also this still to do.